addOns/pscanrules/src/main/java/org/zaproxy/zap/extension/pscanrules/ContentSecurityPolicyScanRule.java

```diff
@@ -481,7 +481,8 @@ private List<String> getAllowedWildcardSources(String policyText) {
         if (checkPolicy(pol::allowsWorker)) {
             allowedSources.add("worker-src");
         }
-        if (checkPolicy(pol::allowsPrefetch)) {
+        if (pol.getFetchDirective(FetchDirectiveKind.PrefetchSrc).isPresent()
+                && checkPolicy(pol::allowsPrefetch)) {
             allowedSources.add("prefetch-src");
         }
```

Comment:
IMO we should just remove this and alert that's deprecated.

Reply:
Okay, I’ll switch this around. So that we handle it for now.

I’ve got an upstream PR that’ll bring that, but who knows how long that’ll take to get merged and released.